debian/changelog

```diff
@@ -1,24 +1,54 @@
-cloud-init (23.3-0ubuntu0~22.04.1) UNRELEASED; urgency=medium
+cloud-init (23.4-0ubuntu0~22.04.1) jammy; urgency=medium
 
   * d/control: add python3-apt as Recommends to read APT config from apt_pkg
-  * d/cloud-init.maintscript: Remove the unused hook-network-manager
-    conffile. (LP: #2027861)
   * d/p/status-do-not-remove-duplicated-data.patch:
     - Revert behavior downstream, leave duplicate data
+  * d/control: Add python3-passlib as needed for testing
```

Comment:
This along with the Akamai and po/templates lines were in 23.3.1 so should be dropped here